### ISSUE TITLE
Please mark as deprecated

### ISSUE THREAD

**Issue by JaredCE** [marcominetti/node-memwatch#51]
Code hasn't been updated since 2017 and doesn't work on latest versions of Node.  Please mark this project as deprecated.

**Comment by Genysys** [comment]
Agreed. use this instead: https://github.com/airbnb/node-memwatch

**Comment by cool-firer** [comment]
@Genysys looks like there is no leak monitor event

**Comment by Venryx** [comment]
> @Genysys looks like there is no leak monitor event

Yes, airbnb's fork removed that part of the code. There explanation was given [here](https://github.com/airbnb/node-memwatch/pull/1#issue-306610771).

And here is the code that was removed that was doing the leak detection: https://github.com/airbnb/node-memwatch/pull/1/files#diff-16ab447da591c718a41d43cb70d0f471069f1db23629b3751d8d7af891b20c13L108

It should be possible to re-implement that behavior in userland JS code. If someone does this transformation (it doesn't look too hard), please post the JS code for it here. :)

**Comment by Venryx** [comment]
Hmm, the airbnb fork for me seems to be failing to build on Node v14.17.1 -- at least on Windows.

Have others gotten it to work on Node v14+, on either Linux or Windows?
